Platform/Radxa/Orion/O6N/Library/PlatformEnvHookLib/PlatformEnvHookLib.c

```diff
@@ -30,6 +30,7 @@ GPIO_CFG  GpioCfgTable[] = {
   { 5,   OUTPUT,            INOUT_LOW,           INTERRUPT_ENABLE_DEFAULT, INTERRUPT_TYPE_DEFAULT }, // M2_WLAN_RST_L
   { 6,   OUTPUT,            INOUT_LOW,           INTERRUPT_ENABLE_DEFAULT, INTERRUPT_TYPE_DEFAULT }, // LOM_RST_L
   { 7,   OUTPUT,            INOUT_LOW,           INTERRUPT_ENABLE_DEFAULT, INTERRUPT_TYPE_DEFAULT }, // PD_RESET
+  { 8,   OUTPUT,            INOUT_LOW,           INTERRUPT_TYPE_DEFAULT,   INTERRUPT_TYPE_DEFAULT }, // GPIO8(DEFAULT LOW, STATE LED ON)

```

Comment:
```suggestion
  { 8,   OUTPUT,            INOUT_LOW,           INTERRUPT_ENABLE_DEFAULT, INTERRUPT_TYPE_DEFAULT }, // GPIO008_STATE_LED
```
目前后面的注释都是原理图里面的net名，方便搜索